onnxruntime/test/contrib_ops/qordered_matmul_op_test.cc

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "qordered_test_utils.h"
+
+namespace onnxruntime {
+namespace test {
+
+// Only the CUDA EP supports ordered quantized ops for now
+#ifdef USE_CUDA
+
+static void RunQOrdered_MatMul_Test(
+    std::vector<int64_t> const& shape_A,
+    std::vector<int64_t> const& shape_B,
+    std::vector<int64_t> const& shape_Y,
+    OrderCublasLt weight_order,
+    float scale_A, float scale_B, float scale_C, float scale_Y,
+    bool add_bias = false, bool broadcast_c_batch = false) {
+  // Needs Turing or higher architecture
+  if (NeedSkipIfCudaArchLowerThan(750)) {
```

Comment:
Can't run these tests on CI as the IMMA MatMul needs Turing or higher arch to run